be/src/storage/persistent_index.h

```diff
@@ -151,7 +163,7 @@ class PersistentIndex {
     Status create(size_t key_size, const EditVersion& version);
 
     // load required states from underlying file
-    Status load();
+    Status load(Tablet* tablet);
```

Comment:
Currently just treat this class as a standalone class, load from a PB object, and commit to a PB object.

Reply:
done